extjsdk/README.md

```diff
@@ -47,6 +46,10 @@ The SDK includes a utility method to retrieve the startup configuration document
 `<connectorWorkingDirector>/serverConfig/server.config`. The SDK will also look for the file in the working connector 
 directory if it is not found in the `serverConfig` subdirectory (i.e. `<connectorWorkingDirector>/server.config`).
 
+In addition to the minimum configuration properties, the `server.config` file can also include the following property:
+*   `sendPings`: A boolean property that, if set to `true`, enables the SDK to send ping messages to the Vantiq Server. 
```

Comment:
To keep all the MD processors happy, usually a good idea to surround bullet lines with blank lines.